client/actions/loginActions.js

```diff
@@ -7,9 +7,7 @@ export function loginUser(credential) {
             .then((response) => {
                 dispatch(loginAction(response.data))
             })
-            .catch((err) => {
-                console.log("error ====>>>", err);
-            })
+            .catch({/** */})
```

Comment:
please don't swallow exceptions.
use this instead
catch((err) => { console.error(err);}